src/main.ts

```diff
@@ -105,37 +132,61 @@ async function getChangedFiles(
   return [changedFiles, !!added]
 }
 
-export async function finish(): Promise<void> {
+async function finish(): Promise<void> {
   try {
+    const tracerPid = core.getState('tracerPid')
+    if (tracerPid) {
+      core.info(`Killing tracer process ${tracerPid}`)
+      child_process.execSync(`sudo kill ${tracerPid}`)
```

Comment:
I know this is a hackathon, but could we convert `tracerPid` to a number before concatenating it with a shell command? It came from grepping through user-controlled data.